src/DataStructures.jl

```diff
@@ -25,7 +25,7 @@ module DataStructures
     export AbstractHeap, compare, extract_all!, extract_all_rev!
     export BinaryHeap, BinaryMinHeap, BinaryMaxHeap, nlargest, nsmallest
     export MutableBinaryHeap, MutableBinaryMinHeap, MutableBinaryMaxHeap
-    export heapify!, heapify, heappop!, heappush!, isheap
+    export heapify!, heapify, heappop!, heappush!, isheap, top
```

Comment:
This is not needed since `@deprecate` exports it already:
```suggestion
    export heapify!, heapify, heappop!, heappush!, isheap
```